native-shared-library/README.md

```diff
@@ -11,22 +11,19 @@ This example demonstrates how to use the **Native Image C API**. We will:
 2. Use the `native-image` tool to create a shared library from the Java class library.
 3. Create and compile a C application that calls an entrypoint method in the shared library.
 
-## Prerequisites
-* [GraalVM](http://graalvm.org)
-* [Native Image](https://www.graalvm.org/latest/reference-manual/native-image/)
-
 ## Preparation
 
 1. Download and install the latest GraalVM JDK with Native Image using the [GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader):
```

Comment:
We need to explain why users need to install `llvm-toolchain`. 
Update the sentence to:

Download and install the latest GraalVM JDK with Native Image and LLVM toolchain using the [GraalVM JDK Downloader](https://github.com/graalvm/graalvm-jdk-downloader). With the LLVM toolchain, you can compile C/C++ code to bitcode using `clang` shipped with GraalVM.